includes/class-wc-stripe-helper.php

```diff
@@ -1496,8 +1496,11 @@ public static function payment_method_allows_manual_capture( string $payment_met
 	 *
 	 * @param WC_Order $order The order.
 	 * @return bool
+	 *
+	 * @deprecated 8.9.0
 	 */
 	public static function is_wallet_payment_method( $order ) {
+		wc_deprecated_function( __FUNCTION__, '8.9.0', 'in_array( $order->get_meta( \'_stripe_upe_payment_type\' ), WC_Stripe_Payment_Methods::WALLET_PAYMENT_METHODS, true )' );
```

Comment:
Small suggestion so that the class name appears in the deprecated notice:
```suggestion
		wc_deprecated_function( __METHOD__, '8.9.0', 'in_array( $order->get_meta( \'_stripe_upe_payment_type\' ), WC_Stripe_Payment_Methods::WALLET_PAYMENT_METHODS, true )' );
```